conformance_tests/tools/metrics/src/test_metric_tracer.cpp

```diff
@@ -1115,4 +1115,178 @@ TEST_F(zetMetricTracerTest,
   }
 }
 
+TEST_F(zetMetricTracerTest,
+       GivenTracerSupportingDevicesThenValidateMetricExportMemory) {
```

Comment:
Shouldn't the name be in the line of "GPU kernel generated event" ? after all that is what the feature is about.